### PR TITLE
Modifications to mapsims for new SO noise simulation

### PR DESCRIPTION
**Pull request by zonca**
Will be tagged 2.3.0

Simulation files:
https://github.com/simonsobs/map_based_simulations/pull/33

### CODE REVIEW

**Comment by msyriac** [comment]
This seems to break the homogenous=True case, since fsky is not defined in Line 738 of noise.py.

File: mapsims/noise.py [hunk 1]
```diff
@@ -38,7 +38,7 @@ def __init__(
         sensitivity_mode="baseline",
         apply_beam_correction=False,
         apply_kludge_correction=True,
-        homogenous=False,
+        homogeneous=False,
```
Comment (zonca) [inline mapsims/noise.py:41]: @msyriac changed spelling of keyword

File: mapsims/noise.py [hunk 2]
```diff
@@ -705,7 +740,7 @@ def simulate(
                 self.get_white_noise_power(tube, sky_fraction=1, units="arcmin2")
                 * nsplits
                 * fsky
-                * wnoise_scale
+                * wnoise_scale.flatten()
```
Comment (zonca) [inline mapsims/noise.py:743]: @msyriac not sure about this fix, please crosscheck

File: mapsims/tests/test_noise.py [hunk 3]
```diff
@@ -47,3 +47,35 @@ def test_noise_simulator_car(tube):
         )
     )
     assert_quantity_allclose(output_map, expected_map, rtol=1e-5)
+
```
Comment (zonca) [inline mapsims/tests/test_noise.py:50]: @msyriac added 2 simple tests here

**Comment by msyriac** [comment]
Please fix the double "pixell.enmap" here: https://github.com/zonca/mapsims/blob/bf397268adafa85b5e25cd701f5291f56174d576/mapsims/noise.py#L752

**Comment by msyriac** [comment]
The self.pmap in https://github.com/zonca/mapsims/blob/bf397268adafa85b5e25cd701f5291f56174d576/mapsims/noise.py#L565
should be self.pixarea_map

**Comment by zonca** [comment]
@msyriac merged it, can you fix doing another PR?

**Pull request merged**